app/javascript/packs/triage_datepicker.js

```diff
@@ -6,4 +6,10 @@ flatpickr(dateElement, {
     dateFormat: "d/m/Y",
     defaultDate: dateElement.value ? null : new Date().fp_incr(6),
     allowInput: true
+});
+
+const contactDobElement = document.getElementById("contact_date_of_birth");
+flatpickr(contactDobElement, {

```

Comment:
Alternatively, you could do `flatpickr('.datepicker')`. I think we already set the default date server-side for the phone triage need anyway (which we probably won't end up doing anymore in the assessment flow anyway).

Reply:
true, that would be nicer.  There are some quirks with how I've built the phone triage need validation etc. once the phone triage has been removed from this 'add-needs' page will simplify